libseccomp/src/filter_context.rs

```diff
@@ -1145,6 +1145,34 @@ impl ScmpFilterContext {
 
         Ok(buf)
     }
+
+    /// Precompute the seccomp filter for future use
+    ///
+    /// This function precomputes the seccomp filter and stores it internally for
+    //  future use, speeding up [`ScmpFilterContext::load()`] and other functions which require
+    //  the generated filter.
```

Comment:
nit: `///`

You might want to check whether the doc is generated properly or not before pushing.

Reply:
My bad, I have fixed it.